testing/bdd/features/timing/ebuttd_resolved_timings_on_elements.feature

```diff
@@ -38,18 +58,20 @@ Feature: Resolving timings on elements
     And   the EBU-TT-Live document is denested
     And   the EBU-TT-Live document is converted to EBU-TT-D
     Then  EBUTTD document is valid
+    And   p resulted begin time is <p_resulted_begin_time>
+    And   p resulted end time is <p_resulted_end_time>
     And   span1 resulted begin time is <span1_resulted_begin_time>
     And   span1 resulted end time is <span1_resulted_end_time>
     And   nestedSpan resulted begin time is <nestedSpan_resulted_begin_time>
     And   nestedSpan resulted end time is <nestedSpan_resulted_end_time>
     And   no timings present on p
```

Comment:
is this needed if the resulted `<p_resulted_begin_time>` & `<p_resulted_end_time>` are None?

Reply:
No I will take those out because the `no timings present on p` step already checks the same thing.

Reply:
Fixed in aeb8da0